.github/workflows/codeql-analysis.yml

```
@@ -0,0 +1,43 @@
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ master ]
+    tags: [ v* ]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [ master ]
+    paths-ignore: [ '**/*.md' ]
+  schedule:
+    - cron: '00 14 * * *' # Every morning at 7:00am PDT
+
+jobs:
+  analyze:
+    name: Analyze
+    runs-on: ubuntu-latest
+    permissions:
+      actions: read
+      contents: read
+      security-events: write
+
+    strategy:
+      fail-fast: false
```

What does this mean?

---

Doesn't fail-fast. It was the default in the template, which I tried to modify as minimally as possible.

---

https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstrategyfail-fast

---

If more than one language were added to the matrix, this ensures that even if one language failed, the rest would continue to run.

---

Looked it up [here](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstrategyfail-fast)

```suggestion
      fail-fast: false # Don't cancel other jobs if one fails
```